doc/source/user/tutorial-svd.rst

```diff
@@ -1,6 +1,6 @@
-================================================
-Tutorial: Linear algebra on n-dimensional arrays
-================================================
+===============================================================
+Tutorial: A tour of n-dimensional arrays
```

Comment:
+1 for tags! I also agree that the original title was more specific, but I'll let others have their say.

Reply:
The original title doesn't sound like a tutorial title -- it sounds like the title of an explanation page. 

I tried out a title like "Compress an image using SVD", but while that's factually true, it doesn't reflect the spirit of the tutorial; SVD is an interesting way to get the user's hands on n-dimensional arrays.

It could be "Do linear algebra on n-dimensional arrays".